elements/nuxeo-admin/nuxeo-repository-analytics.js

```diff
@@ -184,12 +179,7 @@ Polymer({
       </nuxeo-repository-data>
 
       <nuxeo-card heading="[[i18n('repositoryAnalytics.topNCreators.heading', '10')]]">
-        <chart-pie
-          values="[[_values(topCreators)]]"
-          labels="[[_labels(topCreators)]]"
-          options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
-        >
-        </chart-pie>
+        <chart-pie values="[[_values(topCreators)]]" labels="[[_labels(topCreators)]]"> </chart-pie>
```

Comment:
nit: can we remove the extra blank space for consistency?
```suggestion
        <chart-pie values="[[_values(topCreators)]]" labels="[[_labels(topCreators)]]"></chart-pie>
```

Reply:
thanks! done!